FILE packages/app/src/components/voice-compact-indicator.tsx
RANGE {"path":"packages/app/src/components/voice-compact-indicator.tsx","old_lines":[49,53],"new_lines":[49,53],"diff_block_lines":[5,11],"hunk_header":"@@ -49,5 +49,5 @@ const styles = StyleSheet.create((theme) => ({","substantive_changed_lines":2}
@@ -49,5 +49,5 @@ const styles = StyleSheet.create((theme) => ({
     alignItems: "center",
     gap: theme.spacing[2],
-    paddingLeft: theme.spacing[2],
+    paddingLeft: theme.spacing[3],
     paddingRight: theme.spacing[1],
     height: 32,
